homeassistant/components/ezviz/config_flow.py

```diff
@@ -331,7 +336,7 @@ async def async_step_reauth_confirm(
         for item in self._async_current_entries():
             if item.data.get(CONF_TYPE) == ATTR_TYPE_CLOUD:
                 self.context["title_placeholders"] = {ATTR_SERIAL: item.title}
```

Comment:
title_placeholders seems to have no effect on reauth config flow. Can I remove this line? .Doesn't seem to be required.

The title is well labelled  as reauth so I don't have a problem with that.

Reply:
On re-auth the existing title of the config entry will be used (even if the user has renamed it). We don't need to set it.

Reply:
Done. Thanks!